src/vscode-dts/vscode.d.ts

```diff
@@ -20293,6 +20303,56 @@ declare module 'vscode' {
 		constructor(label: string, command: string, args?: string[], env?: Record<string, string | number | null>, version?: string);
 	}
 
+	/**
+	 * McpPackageServerDefinition represents an MCP server available by running
+	 * a local process with the provided package and operating on its stdin and stdout streams.
+	 * The process will be spawned as a child process of the extension host and by default
+	 * will not run in a shell environment.
+	 */
+	export class McpPackageServerDefinition {
+		/**
+		 * The human-readable name of the server.
+		 */
+		readonly label: string;
+
+		/**
+		 * Name of the package that needs to be started.
+		 */
+		readonly packageName: string;
+
+		/**
+		 * The type of the mcp package
+		 */
+		readonly packageType: McpPackageType;
```

Comment:
I think we usually prefer `kind` over `type`. I defer to @mjbvz though. 
```suggestion
		readonly kind: McpPackageKind;
```

Reply:
We also have types - Eg: FileType